diff --git a/pretix/settings.py b/pretix/settings.py
index c871b2b..5101edb 100644
--- a/pretix/settings.py
+++ b/pretix/settings.py
@@ -242,6 +242,7 @@ ENTROPY = {
 PRETIX_EMAIL_NONE_VALUE = 'none@well-known.pretix.eu'
 
 STATIC_ROOT = os.path.join(os.path.dirname(__file__), 'static.dist')
+STATIC_ROOT = os.environ.get('PRETIX_STATIC_ROOT', STATIC_ROOT)
 
 SESSION_COOKIE_NAME = 'pretix_session'
 LANGUAGE_COOKIE_NAME = 'pretix_language'
